### DISCUSSION TITLE
No result after input the command

### DISCUSSION THREAD

**Discussion by networkboxkeith** [Yamato-Security/WELA-deprecated#87] [Q&A] [answered]
Following the Wiki instruction, No output result after. Please help
![Capture](https://user-images.githubusercontent.com/99933882/154602971-a1730659-fdcf-4ab1-b0a2-af388728774a.JPG)

**Comment by hitenkoku** [comment, marked as answer]
@networkboxkeith  Thank you for your comment.

This question is the same as #79.
"-EventIDStatistics" is wrong option. Right option is "-SecurityEventID_Statistics".

README.md was wrong, so I Already updated README.md(#82).
